examples/satellite/main.tf

```diff
@@ -22,9 +22,13 @@ module "tgw-satellite" {
   satellite_destination_cidr_blocks = var.satellite_destination_cidr_blocks
   hub_destination_cidr_blocks       = var.hub_destination_cidr_blocks
 
-  subnet_filters = var.subnet_filters
+  attachment_subnet_filters = var.attachment_subnet_filters
 
   transit_gateway_hub_name = var.transit_gateway_hub_name
 
   route_entire_satellite_vpc = var.route_entire_satellite_vpc
+
+  route_private_subnets_via_tgw = var.route_private_subnets_via_tgw
```

Comment:
Would be great to have a separate test case for this new functionality, while keeping the existing test case in place, to ensure that the old behaviour is still working as expected.

Reply:
@danvaida I added a separate test case